proposal.rst

```diff
@@ -271,7 +309,34 @@ Project Timeline
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+Community Bonding Period (May 4 - May 31)
+    * Interact with mentors, introduce myself to the community.
+    * Get familiar with Xapian’s development style. Understand expected coding, documentation, and testing standards set by Xapian.
+    * Start working on some issues to familiarise with the codebase. (Not necessarily related to the project)
+    * Discuss and prepare a list of various MIME types that need to be worked on.
+    * Read documentation relating to the required libraries.
+
+First Coding Period (June 1 - June 28)
+    * Implement first handler, update build system, and add new worker to omindex. (7 days)
```

Comment:
What is your first handler going to be? Are you thinking libarchive? What file formats would that support? libarchive is unlikely to directly support file formats; you're probably going to have to do some more work, so if that's your preferred first subproject then you should think about the different stages you'll go through to add support for one or more file formats, using libarchive to unpack the zip or similar that the files are stored in.

Reply:
ODT (OpenDocument Text) with help of libarchive will be my first target.